Day6/bitwiseOperators.js

```diff
@@ -0,0 +1,15 @@
+function getMaxLessThanK(n, k) {
+    let max = 0;
+
+    for (let a = 1; a <= n; a++) {
```

Comment:
Nice Implementation, I will suggest that when writing for loop, use "i" for the variable. It's like the "i" is been used across in programming Js. 
With the second for loop, you can use "j". I have seen this implementation in a lot of codes.